includes/embeds/class-amp-youtube-embed-handler.php

```diff
@@ -167,13 +310,7 @@ public function render( $args, $url ) {
 
 		$this->did_convert_elements = true;
 
-		$attributes = array_merge(
-			[ 'data-videoid' => $args['video_id'] ],
-			wp_array_slice_assoc( $args, [ 'layout', 'width', 'height' ] )
-		);
-		if ( ! empty( $args['title'] ) ) {
-			$attributes['title'] = $args['title'];
-		}
+		$args['placeholder'] = ( ! empty( $args['placeholder'] ) ) ? $args['placeholder'] : '';
```

Comment:
The `placeholder` argument would always be set by `parse_props()`, no? Or perhaps it would be better to set the `placeholder` argument here.